pytest.ini

```diff
@@ -3,5 +3,6 @@ addopts = --cov . --pep8 --pylint --cov-report term --cov-report html --ds=micro
 norecursedirs = node_modules .git .tox static templates .* CVS _darcs {arch} *.egg
 pep8ignore =
    */migrations/* ALL
+   */financialaid/*.py W503
```

Comment:
Yes, I don't necessarily agree or disagree but I'd like the linter rules to remain unchanged at least for the duration of the financial aid work, to keep the code consistent with everything else